tests/identities/IdentitiesManager.test.ts

```diff
@@ -16,6 +16,7 @@ import { DB } from '@matrixai/db';
 import { KeyManager } from '@/keys';
 import { IdentitiesManager, providers } from '@/identities';
 import * as identitiesErrors from '@/identities/errors';
+import { generateKeyPair } from '@/keys/utils';
```

Comment:
You should be using `import { utils as keysUtils } from '@/keys` instead.

We only do a direct import of the utility for `./src/utils` or for rare usecases. Everything else should also be be a "qualified import".

Reply:
So inside `tests/keys`. I would use `import * as keysUtils from '...'`.

Outside of that in other `tests/*` I would use `import { utils as keysUtils } from '@/keys';`.

Same situation in `src`.

Reply:
Fixed this.